mitodl/mitxpro
# Add login UI

#### Pre-Flight checklist


#### What are the relevant tickets?
Part of #31 
Fixes #42 

#### What's this PR do?
Adds and integrates login UI with backend

#### How should this be manually tested?
You should be able to login successfully.

NOTE: `LoginPage` is intentionally untested here until we have an easy way of testing `redux-query` connected components (See #61)

#### Screenshots

UI is *very* minimal:

![Screenshot_2019-03-18 mitxpro(1)](https://user-images.githubusercontent.com/28598/54552476-8a861880-4986-11e9-9f8c-0e27b85a9d8f.png)
![Screenshot_2019-03-18 mitxpro](https://user-images.githubusercontent.com/28598/54552477-8a861880-4986-11e9-8cbc-fe0ea983db09.png)
